charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -293,10 +293,12 @@ spec:
             - mountPath: /etc/kubernetes/
               name: azure-cred
             {{- if eq .Values.cloud "AzureStackCloud" }}
```

Comment:
{{- if and (eq .Values.cloud "AzureStackCloud") (ne .Values.linux.distro "fedora") }}